core/types/tx_setcode.go

```diff
@@ -0,0 +1,224 @@
+package types
+
+import (
+	"bytes"
+	"crypto/ecdsa"
+	"errors"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/holiman/uint256"
+)
+
+// DelegationPrefix is used by code to denote the account is delegating to
+// another account.
+var DelegationPrefix = []byte{0xef, 0x01, 0x00}
```

Comment:
What if someone deliberately deploys a contract with such prefix before 7702 is shipped to mainnet, will it fork then?

Reply:
You can't deploy contracts with the `0xEF` prefix before EOF, see https://eips.ethereum.org/EIPS/eip-3541